### PR TITLE
Removed Work Around from Security FVT Servers

### LINKED ISSUE #5016: Remove work-around in LDAPUtils for CONSUL_LDAP_IBM_SECURITY_FVT_SERVICE servers
There were several Security FVT LDAP servers that were not synchronized with each other and they caused FAT failures. I added a work-around starting at line 187 so that the FAT preferred the "correct" LDAP server. The other two servers have now been corrected and this work-around can be removed.

```
                    // TODO Work around LDAP entity sync issues until fixed.
                    ExternalTestService service1 = null, service2 = null, service3 = null;
                    for (ExternalTestService s : services) {
                        if (s.getAddress().contains("7195") || (service2 != null && service3 != null)) {
                            service1 = s;
                        } else if (service2 == null) {
                            service2 = s;
                        } else {
                            service3 = s;
                        }
                    }
```

### PR DESCRIPTION
**Pull request by agiridharan**
Removed temporary work around for IBM LDAP Security FVT server connections that was regulated by Consul. Restored the defaults for the connections.

fixes #5016

### CODE REVIEW

File: dev/fattest.simplicity/src/componenttest/topology/utils/LDAPUtils.java [hunk 1]
```diff
@@ -271,13 +259,42 @@ public Boolean run() {
...
+            Log.info(c, "<clinit>", "LDAP_SERVER_6_NAME=" + LDAP_SERVER_6_NAME);
+            Log.info(c, "<clinit>", "LDAP_SERVER_6_PORT=" + LDAP_SERVER_6_PORT);
+            Log.info(c, "<clinit>", "IBM Continuous Consul LDAP Server");
+            Log.info(c, "<clinit>", "LDAP_SERVER_1_NAME=" + LDAP_SERVER_1_NAME);
+            Log.info(c, "<clinit>", "LDAP_SERVER_1_PORT=" + LDAP_SERVER_1_PORT);
+            Log.info(c, "<clinit>", "LDAP_SERVER_5_NAME=" + LDAP_SERVER_5_NAME);
+            Log.info(c, "<clinit>", "LDAP_SERVER_5_PORT=" + LDAP_SERVER_5_PORT);
+            Log.info(c, "<clinit>", "IBM Security FVT Consul LDAP Server");
+            Log.info(c, "<clinit>", "LDAP_SERVER_4_NAME=" + LDAP_SERVER_4_NAME);
+            Log.info(c, "<clinit>", "LDAP_SERVER_4_PORT=" + LDAP_SERVER_4_PORT);
```
Comment (jvanhill) [inline dev/fattest.simplicity/src/componenttest/topology/utils/LDAPUtils.java]: Add line for LDAP_SERVER_4_SSL_PORT

File: dev/fattest.simplicity/src/componenttest/topology/utils/LDAPUtils.java [hunk 2]
```diff
@@ -271,13 +259,42 @@ public Boolean run() {
...
+            Log.info(c, "<clinit>", "LDAP_SERVER_7_NAME=" + LDAP_SERVER_7_NAME);
+            Log.info(c, "<clinit>", "LDAP_SERVER_7_PORT=" + LDAP_SERVER_7_PORT);
```
Comment (jvanhill) [inline dev/fattest.simplicity/src/componenttest/topology/utils/LDAPUtils.java]: Add line for LDAP_SERVER_7_SSL_PORT

File: dev/fattest.simplicity/src/componenttest/topology/utils/LDAPUtils.java [hunk 3]
```diff
@@ -271,13 +259,42 @@ public Boolean run() {
...
+            Log.info(c, "<clinit>", "LDAP_SERVER_8_NAME=" + LDAP_SERVER_8_NAME);
+            Log.info(c, "<clinit>", "LDAP_SERVER_8_PORT=" + LDAP_SERVER_8_PORT);
```
Comment (jvanhill) [inline dev/fattest.simplicity/src/componenttest/topology/utils/LDAPUtils.java]: Add line for LDAP_SERVER_8_SSL_PORT

File: dev/fattest.simplicity/src/componenttest/topology/utils/LDAPUtils.java [hunk 4]
```diff
@@ -271,13 +259,42 @@ public Boolean run() {
...
+            Log.info(c, "<clinit>", "IBM Security Consul LDAP Server");
+            Log.info(c, "<clinit>", "LDAP_SERVER_10_NAME=" + LDAP_SERVER_10_NAME);
+            Log.info(c, "<clinit>", "LDAP_SERVER_10_PORT=" + LDAP_SERVER_10_PORT);
+            Log.info(c, "<clinit>", "LDAP_SERVER_12_NAME=" + LDAP_SERVER_12_NAME);
+            Log.info(c, "<clinit>", "LDAP_SERVER_12_PORT=" + LDAP_SERVER_12_PORT);
+            Log.info(c, "<clinit>", "Oracle SVT Consul LDAP Server");
+            Log.info(c, "<clinit>", "LDAP_SERVER_3_NAME=" + LDAP_SERVER_3_NAME);
+            Log.info(c, "<clinit>", "LDAP_SERVER_3_PORT=" + LDAP_SERVER_3_PORT);
+            Log.info(c, "<clinit>", "LDAP_SERVER_13_NAME=" + LDAP_SERVER_13_NAME);
+            Log.info(c, "<clinit>", "LDAP_SERVER_13_PORT=" + LDAP_SERVER_13_PORT);
```
Comment (jvanhill) [inline dev/fattest.simplicity/src/componenttest/topology/utils/LDAPUtils.java]: Add line for LDAP_SERVER_13_SSL_PORT

File: dev/fattest.simplicity/src/componenttest/topology/utils/LDAPUtils.java [hunk 5]
```diff
@@ -271,13 +259,42 @@ public Boolean run() {
...
+            Log.info(c, "<clinit>", "LDAP_SERVER_8_NAME=" + LDAP_SERVER_8_NAME);
+            Log.info(c, "<clinit>", "LDAP_SERVER_8_PORT=" + LDAP_SERVER_8_PORT);
+            Log.info(c, "<clinit>", "IBM Security Consul LDAP Server");
+            Log.info(c, "<clinit>", "LDAP_SERVER_10_NAME=" + LDAP_SERVER_10_NAME);
+            Log.info(c, "<clinit>", "LDAP_SERVER_10_PORT=" + LDAP_SERVER_10_PORT);
+            Log.info(c, "<clinit>", "LDAP_SERVER_12_NAME=" + LDAP_SERVER_12_NAME);
+            Log.info(c, "<clinit>", "LDAP_SERVER_12_PORT=" + LDAP_SERVER_12_PORT);
+            Log.info(c, "<clinit>", "Oracle SVT Consul LDAP Server");
+            Log.info(c, "<clinit>", "LDAP_SERVER_3_NAME=" + LDAP_SERVER_3_NAME);
+            Log.info(c, "<clinit>", "LDAP_SERVER_3_PORT=" + LDAP_SERVER_3_PORT);
```
Comment (jvanhill) [inline dev/fattest.simplicity/src/componenttest/topology/utils/LDAPUtils.java]: Add line for LDAP_SERVER_3_SSL_PORT

**Comment by agiridharan** [comment]
#build

**Pull request merged**